src/main/java/com/scottlogic/Main.java

```diff
@@ -21,7 +21,10 @@ public static void main(String[] args) {
                 "An example of a post \nwith lines breaks.", 3);
 
         List<UserPost> userPosts = Arrays.asList(userPost1, userPost2, userPost3);
+        AuthorPostSorter aps = new AuthorPostSorter();
+        aps.sort(userPosts, SortOrder.ASC);
```

Comment:
I think we were going to remove this?

Reply:
I did.. don't know why its not reflected here. This is how I have it now "AuthorPostSorter sortByAuthor = new AuthorPostSorter();"
Then today, I've made it private as well